native/1.3.0/README.md

```diff
@@ -0,0 +1,25 @@
+# Native Library 1.3,0
+
+* Source: https://github.com/rsksmart/native
+* Tag: `1.3.0`
+
+## Build
+
+```
+$ docker build --build-arg NATIVE_LIB_VERSION=1.3.0 -t native-1.3.0 .
```

Comment:
```suggestion
$ docker build -t native-1.3.0 .
```
seems there's no need to pass the arg. Its value is already defined in Dockerfile